.github/workflows/ci.yaml

```diff
@@ -0,0 +1,93 @@
+name: '[CI]'
+run-name: "${{ inputs.run_name }}"
+on:
+  workflow_call:
+    inputs:
+      git_sha:
+        required: true
+        type: string
+      run_name:
+        required: true
+        type: string
+      resource_id:
+        required: true
+        type: string
+      goal:
+        required: false
+        type: string
+        default: "test"
+      os_distros:
+        description: 'Operating System Distributions (comma-separated, e.g., al2,al2023)'
+        default: "al2,al2023"
+        required: false
+        type: string
+      k8s_versions:
+        description: 'Kubernetes Versions (comma-separated, e.g., 1.29,1.30)'
+        default: "1.24,1.25,1.26,1.27,1.28,1.29,1.30,1.31"
```

Comment:
any easily way to get rid of the duplicate parameters in here and `ci-manual`?

Reply:
I think they can be removed from the bot-driven workflow. I’ll open a follow up to do some renaming and cleanup